--- origsrc/frei0r-plugins-1.1.22/src/Makefile.am	2008-10-28 06:17:10.000000000 -0500
+++ src/frei0r-plugins-1.1.22/src/Makefile.am	2009-11-19 23:51:00.829527800 -0600
@@ -119,7 +119,7 @@ RGB_la_SOURCES = mixer3/RGB/RGB.c
 
 AM_CPPFLAGS = -I@top_srcdir@/include 
 AM_CFLAGS = -I@top_srcdir@/include 
-AM_LDFLAGS = -module -avoid-version -Wc,-nostartfiles
+AM_LDFLAGS = -module -avoid-version -no-undefined -shrext .so
 
 plugindir = @prefix@/lib/frei0r-1
 
